internal/controller/instaslice_controller_test.go

```diff
@@ -43,6 +46,54 @@ import (
 	"github.com/openshift/instaslice-operator/internal/controller/utils"
 )
 
+var _ = Describe("Instaslice Predicate", func() {
```

Comment:
I don't think these cases cover the original functionality that is part of `instaslice_controller.go`
There is also a minor update needed to the existing test cases i.e. Expecting the labels for a pod that is created, etc.